stm/stm.ml

```diff
@@ -1005,25 +1004,12 @@ let stm_vernac_interp ?proof ?route id st { verbose; loc; expr } : Vernacstate.t
      to design the stm command interpreter now *)
   set_id_for_feedback ?route dummy_doc id;
   Aux_file.record_in_aux_set_at ?loc ();
-  (* We need to check if a command should be filtered from
-   * vernac_entries, as it cannot handle it. This should go away in
-   * future refactorings.
-   *)
-  let rec is_filtered_command = function
-  | VernacResetName _ | VernacResetInitial | VernacBack _
-  | VernacBackTo _ | VernacRestart | VernacUndo _ | VernacUndoTo _
-  | VernacBacktrack _ | VernacAbortAll | VernacAbort _ -> true
```

Comment:
This looks wrong, all the undo commands should still be filtered. IMHO even if not optimal, the current behavior is better than the abort this would produce.